build-tools/automation/azure-pipelines-release-trigger.yaml

```diff
@@ -0,0 +1,45 @@
+# Xamarin.Android Release Definition Trigger
+
+name: $(Build.SourceBranchName)-$(Build.SourceVersion)-$(Rev:r)
+
+trigger: none
+pr: none
+
+resources:
+  pipelines:
+  - pipeline: xamarin-android
+    source: Xamarin.Android
+    trigger:
+      stages:
+      - post_build
+
+jobs:
+- job: release_trigger
+  displayName: Upload Info from Triggering Build
+  timeoutInMinutes: 30
+  pool: VSEngSS-MicroBuild2022-1ES
+  workspace:
+    clean: all
+  steps:
+  - checkout: none
+
+  - powershell: |
+      $json = @"
+      {
+          "triggeringbuild": {
+              "id": "$(Build.TriggeredBy.BuildId)",
+              "pipeline": "$(Build.TriggeredBy.DefinitionId)",
+              "project": "$(Build.TriggeredBy.ProjectID)"
+          }
+      }
+      "@
+      New-Item -Path $(Build.StagingDirectory) -Name triggering-build.json -ItemType file -Value $json -Force
+      $jsonContent = Get-Content "$(Build.StagingDirectory)\triggering-build.json" | ConvertFrom-Json
+      Write-Host Triggering Build Info: $jsonContent.triggeringbuild.project - $jsonContent.triggeringbuild.pipeline - $jsonContent.triggeringbuild.id
+    displayName: write triggering-build.json
+
+  - task: PublishPipelineArtifact@1
+    displayName: upload triggering-build.json
+    inputs:
+      artifactName: triggering-build.json

```

Comment:
`artifactName` seems a bit too specific here. Plus a bit odd having an extension name. Perhaps call it "build-info" or "triggering-build-info" in case you want include other stuff in the artifact down the road

Reply:
Done